### PR TITLE
Add etcd-replicators resource info to RBAC and sensuctl references

### PR DESCRIPTION
**Pull request by hillaryfraley**
- Adds etcd-replicators as an RBAC cluster-wide resource
- Adds EtcdReplicators as a `sensuctl create` resource type

Additional changes required to document etcd-replicators (federation)
See https://github.com/sensu/sensu-docs/pull/1829

### CODE REVIEW

File: content/sensu-go/5.15/reference/rbac.md [hunk 1]
```diff
@@ -197,6 +197,7 @@ Cluster-wide resources cannot be assigned to a namespace and can only be accesse
 | `cluster`   | Sensu clusters running multiple [Sensu backends][1] |
 | `clusterrolebindings`   | Cluster-wide role assigners  |
 | `clusterroles`   | Cluster-wide permission sets  |
+| `etcd-replicators` | [Mirror RBAC resource changes][40] to follower clusters |
```
Comment (hillaryfraley) [inline content/sensu-go/5.15/reference/rbac.md:200]: Need to confirm this is the correct resource type. Or should it be `etcdreplicators` here?

File: content/sensu-go/5.15/sensuctl/reference.md [hunk 2]
```diff
@@ -284,12 +284,13 @@ cat my-resources.yml | sensuctl create
 `AdhocRequest` | `adhoc_request` | `Asset` | `asset`
 `CheckConfig` | `check_config` | `ClusterRole`  | `cluster_role`
 `ClusterRoleBinding`  | `cluster_role_binding` | `Entity` | `entity`
-`Event` | `event` | `EventFilter` | `event_filter`
-`Handler` | `handler` | `Hook` | `hook`
-`HookConfig` | `hook_config` | `Mutator` | `mutator`
-`Namespace` | `namespace` | `Role` | `role`
-`RoleBinding` | `role_binding` | `Silenced` | `silenced`
-[`ldap`][26] | [`ad`][26] | [`TessenConfig`][27] | [`PostgresConfig`][32]
+[`EtcdReplicators`][35] | `Event` | `event` | `EventFilter`
```
Comment (hillaryfraley) [inline content/sensu-go/5.15/sensuctl/reference.md:291]: Again, need to confirm `EtdcReplicators` is correct here.

**Comment by hillaryfraley** [comment]
@echlebek following up on this PR

File: content/sensu-go/5.15/reference/rbac.md (hunk 1 above)
Comment (echlebek) [reply content/sensu-go/5.15/reference/rbac.md:200]: This is correct

File: content/sensu-go/5.15/sensuctl/reference.md (hunk 2 above)
Comment (echlebek) [reply content/sensu-go/5.15/sensuctl/reference.md:291]: Confirmed

**Review by echlebek — APPROVED** [review]
LGTM!

**Pull request merged**